src/libstd/sys/unix/fs.rs

```diff
@@ -25,6 +24,10 @@ use sys::platform::raw;
 use sys::{cvt, cvt_r};
 use sys_common::{AsInner, FromInner};
 use vec::Vec;
+#[cfg(target_os = "solaris")]
+use core_collections::borrow::ToOwned;
```

Comment:
I think you can remove the `#[cfg]` and the one below by just adding `use prelude::v1::*` to the top of this file (these are both in the prelude), and then you don't need the `#[cfg]` as well I believe.